### DISCUSSION TITLE
How to set up Icon for clear button inside Select's Dropdown Menu?

### DISCUSSION THREAD

**Discussion by alexAdtronics** [olton/Metro-UI-CSS#1937] [Q&A] [answered]
Before v5.0.4, that icon used to be ```"<span class='default-icon-cross'></span>" ```, and after v5.0.4, it became a '❌', is it possible to change it to other icons?

**Comment by olton** [comment]
In the `5.0.6` (current dev branch) or CDN `.../dev/...` you can use an attribute `data-clear-button-icon` to change it.

**Reply by alexAdtronics** [reply]
That can only change the Select's data-clear-button icon, how about Select's Dropdown's search bar's data-clear-button?

![image](https://github.com/olton/Metro-UI-CSS/assets/149552010/83fc854d-c9eb-463a-bc34-d91abb88fe20)

**Comment by olton** [comment, marked as answer]
One attribute for both
